dynawo/3rdParty/CMakeLists.txt

```diff
@@ -465,9 +474,20 @@ endif()
 # find Boost in two steps to avoid warnings 'Imported targets and dependency information not available for Boost version'
 find_package(Boost ${boost_version} QUIET)
 if(Boost_FOUND)
-  find_package(Boost ${boost_version} QUIET COMPONENTS filesystem program_options serialization system log iostreams)
+  find_package(
+    Boost ${boost_version} QUIET
+    COMPONENTS                #     Dynawo      3rdParty      Powsybl
+      date_time               #                                 X
+      filesystem              #                   X             X
+      program_options         #                   X
+      serialization           #       X           X
+      system                  #                   X             X
+      log                     #       X           X
+      log_setup               #       X
+      iostreams               #                   X
+      unit_test_framework     #                                 X
```

Comment:
Shouldn't we add the PowSybl dependencies only in c++11?

Reply:
I choosed simpliest way to bring the modification.

Reply:
I add a little test and an if finally